design/ElectroBlade Cards.md

```diff
@@ -0,0 +1,30 @@
+##ElectroBlade Cards:
+
+####All Cards can be played to generate 2 Energy.
+
+Jolt - 1 Energy(Tier 1)
+Deal 1x damage and Stun
```

Comment:
Guaranteed stuns for 1 Energy? That sounds insanely powerful.

Even the HoG version with Flinch having a % chance was over-curve.